clients/privacy-center/app/server-environment.ts

```diff
@@ -264,6 +271,9 @@ export const loadPrivacyCenterEnvironment =
       IS_OVERLAY_ENABLED: process.env.FIDES_PRIVACY_CENTER__IS_OVERLAY_ENABLED
         ? process.env.FIDES_PRIVACY_CENTER__IS_OVERLAY_ENABLED === "true"
         : false,
+      IS_PREFETCH_ENABLED: process.env.FIDES_PRIVACY_CENTER__IS_PREFETCH_ENABLED
+        ? process.env.FIDES_PRIVACY_CENTER__IS_PREFETCH_ENABLED === "true"
+        : true,
```

Comment:
I'd play it safe here - there's enough things that can go wrong that we should default this to false:
```suggestion
      IS_PREFETCH_ENABLED: process.env.FIDES_PRIVACY_CENTER__IS_PREFETCH_ENABLED
        ? process.env.FIDES_PRIVACY_CENTER__IS_PREFETCH_ENABLED === "true"
        : false,
```